src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/WebClientConfig.kt

```diff
@@ -101,7 +101,8 @@ class WebClientConfig(
     oAuth2AuthorizedClientService: OAuth2AuthorizedClientService?
   ): WebClient? = getClientCredsWebClient(
     "$communityApiUrl/secure",
-    authorizedClientManagerAppScope(clientRegistrationRepository, oAuth2AuthorizedClientService)
+    authorizedClientManagerAppScope(clientRegistrationRepository, oAuth2AuthorizedClientService),
+    "community-api",
```

Comment:
Client managers are saved in a repository keyed by registration ID. Community-api needs its own client manager because it needs the username to NOT be on the token.